AI/Nullkiller/AIGateway.cpp

```diff
@@ -828,7 +828,10 @@ void AIGateway::makeTurn()
 	boost::shared_lock<boost::shared_mutex> gsLock(CGameState::mutex);
 	setThreadName("AIGateway::makeTurn");
 
-	cb->sendMessage("vcmieagles");
+	if(cb->getStartInfo()->extraOptionsInfo.cheatsAllowed)
+		cb->sendMessage("vcmieagles");
+	else
+		cb->sendMessage("Nullkiller AI currently requires the ability to cheat in order to function correctly! Please enable!");
```

Comment:
Will this result in multiple messages from each AI on every turn?

Reply:
One message per turn per AI player. Yes. but it's still better than cryptic "vcmieagles" message (which also appears in the same interval).

And I think it's better, than completly silent it (as the AI may not work as intended).

Reply:
Why not display it once per game?